### DISCUSSION TITLE
when can we get the keycloak v17.x on dockerhub

### DISCUSSION THREAD

**Discussion by siyueshiqi** [keycloak/keycloak#11235] [Q&A] [answered]
We can only get the keycloak v16.x on docke rhub. https://hub.docker.com/r/jboss/keycloak/tags
But on Quay.io, we can get keycloak v17.x. https://quay.io/repository/keycloak/keycloak?tab=tags
So, may I know when can we get the keycloak v17.x on docker hub?

**Comment by abstractj** [comment, marked as answer]
@siyueshiqi please refer to https://hub.docker.com/r/keycloak/keycloak/tags, all the images are available now.

**Reply by siyueshiqi** [reply]
Thanks.

**Reply by siyueshiqi** [reply]
But I still have one concern. Why the account was switched from **JBoss** to **Keycloak**?

- JBoss Account: https://hub.docker.com/r/jboss/keycloak/tags
- Keycloak Account: https://hub.docker.com/r/keycloak/keycloak/tags

**Reply by senyahnoj** [reply]
It would be nice to put a prominent note on the jboss/keycloak docker hub page explaining that you've done this.

I've spent a couple of months waiting to see the 17/18 updates appear there and it's only today I discovered this post :(

**Comment by abstractj** [comment]
@siyueshiqi we switched to have all the images under the Keycloak organization. So, we have more autonomy over it.

**Reply by siyueshiqi** [reply]
> @siyueshiqi we switched to have all the images under the Keycloak organization. So, we have more autonomy over it.

May I know the differences of keycloak images between Docker Hub and Quay IO? And are there any limitations of pulling images from Quay IO for anonymous usage?

**Reply by abstractj** [reply]
@siyueshiqi those images are built in the same way, the only difference is the fact that they are pushed to 2 different channels: Quay.io and Docker Hub. I'm not aware about any limitations for anonymous usage.

**Reply by siyueshiqi** [reply]
@abstractj Thanks for your help and your information.
